### PR TITLE
Sec cell timer

### PR DESCRIPTION
**Pull request by Afya212**
Door timers now tell you the amount of time remaining on examine, as it is hard to read placed within a window.
Fixed the description of i116 in the sentencing console.

### CODE REVIEW

**Comment by Afya212** [comment]
!review

File: html/changelogs/Afya212-Sec-Fixes.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Cell timers now show time remaining in description."
+  - bugfix: "Fixed i116 in sentencing consoles."
```
Comment (Ben10083) [inline html/changelogs/Afya212-Sec-Fixes.yml:42]:
```suggestion
  - spellcheck: "Fixed i116 description in sentencing consoles."
```

Its semantics but eh, should be spellcheck

**Review by Ben10083 — CHANGES_REQUESTED** [review]

File: html/changelogs/Afya212-Sec-Fixes.yml (hunk 1 above)
Comment (Afya212) [reply html/changelogs/Afya212-Sec-Fixes.yml:42]: Nein

File: code/modules/law/laws/low_severity.dm [hunk 2]
```diff
@@ -157,7 +157,7 @@
 
 /datum/law/low_severity/necrophilia
 	name = "i116 - Disrespect to the Dead"
-	desc = "To threaten to kill or seriously injure an employee."
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:160]: The law datums are only relevant if we arnt pulling the "laws" from the db.
Which we are. So the law datums are irrelevant.

Comment (Afya212) [reply code/modules/law/laws/low_severity.dm:160]: Then the DB is also incorrect as it has this same issue. Regardless it fixes something in the code.

**Review by JohnWildkins — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**